pihole

```diff
@@ -180,7 +185,7 @@ helpFunc() {
 ::: Control all PiHole specific functions!
 :::
 ::: Usage: pihole [options]
-:::		Add -h after -w (whitelist), -b (blacklist), or -c (chronometer)  for more information on usage
+:::		Add -h after -w (whitelist), -b (blacklist), -c (chronometer), or -web (webpage)  for more information on usage
```

Comment:
Mind if I commit a change to make this `-a` for `admin` and avert the possibility of mixing up `-w` on accident?